expression/aggregation/window_func.go

```diff
@@ -70,6 +70,22 @@ var noFrameWindowFuncs = map[string]struct{}{
 	ast.WindowFuncRowNumber:   {},
 }
 
+var useDefaultFrameWindowFuncs = map[string]ast.FrameClause{
+	ast.WindowFuncRowNumber: {
+		Type: ast.Rows,
+		Extent: ast.FrameExtent{
+			Start: ast.FrameBound{Type: ast.CurrentRow},
+			End:   ast.FrameBound{Type: ast.CurrentRow},
+		},
+	},
+}
+
+// UseDefaultFrame indicates if the window function has a provided frame that will override user's designation
+func UseDefaultFrame(name string) (bool, ast.FrameClause) {
+	frame, ok := useDefaultFrameWindowFuncs[strings.ToLower(name)]
+	return ok, frame
+}
+
```

Comment:
why row_number need default frame and others needn't?
it may block func grouping just like:
```sql
mysql> explain select *, row_number() over w, sum(salary) over w from employee window w as (partition by deptid);
+--------------------------------+---------+-----------+----------------+---------------------------------------------------------------------------------------------------------+
| id                             | estRows | task      | access object  | operator info                                                                                           |
+--------------------------------+---------+-----------+----------------+---------------------------------------------------------------------------------------------------------+
| Projection_8                   | 17.00   | root      |                | test.employee.empid, test.employee.deptid, test.employee.salary, Column#8, Column#7                     |
| └─Window_10                    | 17.00   | root      |                | row_number()->Column#8 over(partition by test.employee.deptid rows between current row and current row) |
|   └─Window_11                  | 17.00   | root      |                | sum(test.employee.salary)->Column#7 over(partition by test.employee.deptid)                             |
|     └─Sort_15                  | 17.00   | root      |                | test.employee.deptid                                                                                    |
|       └─TableReader_14         | 17.00   | root      |                | data:TableFullScan_13                                                                                   |
|         └─TableFullScan_13     | 17.00   | cop[tikv] | table:employee | keep order:false, stats:pseudo                                                                          |
+--------------------------------+---------+-----------+----------------+---------------------------------------------------------------------------------------------------------+
6 rows in set (0.00 sec)
```